sql/src/main/java/org/apache/druid/sql/calcite/parser/SqlGranularityLiteral.java

```diff
@@ -54,13 +54,6 @@ public SqlGranularityLiteral clone(SqlParserPos pos)
     return new SqlGranularityLiteral(granularity, unparseString, pos);
   }
 
-  @Override
-  @Deprecated
-  public Object clone()
```

Comment:
Thought overriding a deprecated method for the sake of throwing an exception was not really needed.